### DISCUSSION TITLE
Is saving filters in grid made available from Pimcore 11?

### DISCUSSION THREAD

**Discussion by prunepal333** [pimcore/pimcore#13999] [Q&A / Support] [answered]
I looked at the recent pimcore 11, I see save filters checkbox available in save and share tab of grid config dialog box.
Is it only available for Pimcore 11? Is there any possible to have it on Pimcore X?
![screenshotsvf](https://user-images.githubusercontent.com/59438412/210952019-475c5393-0e57-418f-b33a-26210ecfdeec.png)

**Comment by SamyMP** [comment, marked as answer]
It is only available on Pimcore 11 🙂

If I understood well, Pimcore 10.5.x will only focus on bug fix, so any new feature like this one has been merged in Pimcore 11 only

**Reply by dvesh3** [reply]
This PR introduces the change https://github.com/pimcore/pimcore/pull/13498
